{{ cookiecutter.project_shortname }}/README.rst

```diff
@@ -18,10 +18,9 @@
 .. image:: https://img.shields.io/github/license/{{ cookiecutter.github_repo }}.svg
         :target: https://github.com/{{ cookiecutter.github_repo }}/blob/master/LICENSE
 
-
-{{ cookiecutter.description }}
-
 *This is an experimental developer preview release.*
 
-* Free software: GPLv2 license
-* Documentation: https://{{ cookiecutter.project_shortname }}.readthedocs.io/
+TODO: Please provide feature overview of module
```

Comment:
The idea is that each module should provide a more detailed description of its features (i.e. some few bullets) - see e.g. [Invenio-Admin's README.rst](https://github.com/inveniosoftware/invenio-admin) for an example.